repository-data/webfiles/src/main/resources/site/scss/nhsuk.scss

```diff
@@ -112,3 +112,10 @@
 @import "components/callout-box";
 @import "components/link-list";
 @import "components/utils";
+
+// Hide the PDF button if JS is off
+.no-js {
```

Comment:
Is this the best file to put this? Wouldn't it be better placed in `nhsuk-print-pdf-document.scss` with something like the following: `.no-js & {
  display: none;
}`

Reply:
This is a fix for screen not printing, and it's the PDF printing button specifically we want to hide.

Reply:
I moved the definition into the `components/utils` file to keep the main sass file clean.